client/src/blockchain/abiAddressMapping.ts

```diff
@@ -0,0 +1,9 @@
+import ProductionMappings from '../temp/ProductionMappings.json';
+import DevMappings from '../temp/BehodlerABIAddressMappingDev.json';
+import RopstenMappings from '../temp/BehodlerABIAddressMappingRopsten.json';
+
+export const getBehodlerABIAddressMapping = (networkName: string) => ({
+    main: ProductionMappings,
+    development: DevMappings,
+    ropsten: RopstenMappings,
+}[networkName] || ProductionMappings);
```

Comment:
This is a helper function created to dynamically get the correct "mappings" file depending on the network we use. I've created it only to avoid additionally bloating the logic behind setting up the contracts in ethereumAPI.ts.